src/citra_qt/configuration/configure_audio.cpp

```diff
@@ -127,6 +127,17 @@ void ConfigureAudio::updateAudioInputDevices(int index) {
                                            !Core::System::GetInstance().IsPoweredOn());
 }
 
+void ConfigureAudio::updateAudioDevices(int sink_index) {
```

Comment:
This was meant to be removed by renaming `updateAudioDevices` to `updateAudioOutputDevices` since we now have output and input audio devices

Reply:
ahh i see. I'll fix it right away!